src/test/java/org/snomed/snowstorm/core/data/services/BranchMergeServiceTest.java

```diff
@@ -3574,6 +3597,396 @@ void assertReleaseProperties(
 		}
 	}
 
+	@Test
+	void testRebasingSynonymAdditions() {
```

Comment:
This tests there are no regressions, i.e. adding multiple synonyms produce no conflicts.

Reply:
@dmcgihtsdo Can we move these tests via rest template to the controller level please?

Reply:
I'll try @CoderMChu 

I think I did it via rest as no conflicts were returning... but I might be confused with a different test / scenario so I'll give it a go.

Reply:
I have updated the tests @CoderMChu to be controller level.